### ISSUE TITLE
[MA0184] False positive on empty string

### ISSUE THREAD

**Issue by MaceWindu** [meziantou/Meziantou.Analyzer#1000]
### Version of the Meziantou.Analyzer NuGet package

2.0.298

### Rule Identifier

MA0184

### Target Framework

net10.0

### C# Language version

C# 14

### Description

I have an API, that accepts only interpolation handler (no string-based overloads).

```cs
public static void MyAPI([InterpolatedStringHandlerArgument] ref MyInterpolatedStringHandler handler)
{
}
```

There are few cases where I need to pass plain string to it which I do using `MyAPI($"something")` and it doesn't trigger MA0184 as expected. But for `MyAPI($"")` call I'm getting analyzer error.

### Reproduction Steps

Minimal code:

```c#
code to reproduce the error
```
